sample-project/app/controllers/users_controller.rb

```diff
@@ -63,7 +63,7 @@ def destroy
 
   private
     # Use callbacks to share common setup or constraints between actions.
-    def set_user
+    def set_user2
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vividmuimui/danger-sample/blob/52cb3991be8415173092296f7c5dd1efec42d2b1/sample-project/app/controllers/users_controller.rb#L66"></span>Layout/IndentationWidth: Use 2 (not 4) spaces for indentation.</td>
    </tr>
  </tbody>
</table>